### DISCUSSION TITLE
Will server restart the next time Ubuntu finishes loading after docker shutdown command?

### DISCUSSION THREAD

**Discussion by Mathiaszero** [appwrite/appwrite#4636] [Q&A] [answered]
Hello, I am planning to use the command "docker-compose down".  When I restart Ubuntu and it loads back up again, will Appwrite automatically boot again, or do I have to use "docker-compose up" every time? I don't want to ruin the Appwrite configuration so I have not tried it yet.

**Comment by stnguyen90** [comment, marked as answer]
No, Appwrite won't start up again automatically and you'll have to run `docker compose up -d`. You could look into running `docker compose up -d` automatically though. How to do it varies by operating system.

I do suggest running `docker compose down` before shutting down your server, though. There is a chance data can be corrupted if mariadb is not cleanly shut down.
